### PR TITLE
TextArea: add a Getting Started tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/Form/15 Configure Item Labels/05 Location and Alignment/05 Align Labels Relatively to Editors.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ The Form UI component displays labels on the left side of their editors and alig
 
 ---
 
-A label placed on the left or right side of the editor is centered vertically in most cases. The labels of the editors that occupy much screen space like the [Calendar](/api-reference/10%20UI%20Components/dxCalendar '/Documentation/ApiReference/UI_Components/dxCalendar/'), [TextArea](/concepts/05%20UI%20Components/TextArea/00%20Overview.md '/Documentation/Guide/UI_Components/TextArea/Overview/'), and [RadioGroup](/concepts/05%20UI%20Components/RadioGroup/00%20Overview.md '/Documentation/Guide/UI_Components/RadioGroup/Overview/') are aligned at the top, but you can center them using the following code:
+A label placed on the left or right side of the editor is centered vertically in most cases. The labels of the editors that occupy much screen space like the [Calendar](/api-reference/10%20UI%20Components/dxCalendar '/Documentation/ApiReference/UI_Components/dxCalendar/'), [TextArea](/Documentation/ApiReference/UI_Components/dxTextArea/), and [RadioGroup](/concepts/05%20UI%20Components/RadioGroup/00%20Overview.md '/Documentation/Guide/UI_Components/RadioGroup/Overview/') are aligned at the top, but you can center them using the following code:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Form/15 Configure Item Labels/05 Location and Alignment/05 Align Labels Relatively to Editors.md]: I felt the urge to add some structure - a list or a table - to this. The information lends itself to this format... 
I could also add a third column with control examples. 


```suggestion
If a label's position is left or right, the label's vertical alignment depends on the control type:

| Control Type | Default Vertical Alignment |
| ------------- | ----------------|
| Single-line editors | Middle |
| Multi-line editors and bigger controls | Top |

The code below changes the default vertical alignment for a label:
```

File: concepts/05 UI Components/Form/15 Configure Item Labels/05 Location and Alignment/05 Align Labels Relatively to Editors.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ The Form UI component displays labels on the left side of their editors and alig
...
 
     <!--CSS-->
     .dx-label-h-align {
```
Comment (Abadzhev) [inline concepts/05 UI Components/Form/15 Configure Item Labels/05 Location and Alignment/05 Align Labels Relatively to Editors.md:155]: The code says ".dx-label-h-align" - doesn't it mean "horizontal alignment"? But it should set vertical alignment, if I understand the text correctly.

File: concepts/05 UI Components/TextArea/00 Getting Started with TextArea/10 Specify Value and Label.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+To set an initial TextArea value, use the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property. You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters users can enter. Note that this property only limits the number of characters for users. You can enter text that exceeds the maximum character length programmatically, but the number of characters displayed to users will still be limited by this property setting.
```
Comment (Abadzhev) [inline concepts/05 UI Components/TextArea/00 Getting Started with TextArea/10 Specify Value and Label.md]: If we're talking about setting the value programmatically (in code) I wouldn't use the verb "enter". 

```suggestion
To set an initial TextArea value, use the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property. You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters users can enter. Note that this property only limits the number of characters for users. Your code can specify text that exceeds the maximum character length, but the number of characters displayed to users will still be limited by this property setting.
```

File: concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+If the size of the UI component should be fixed, specify the [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#width) properties.
```
Comment (Abadzhev) [inline concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md]: A minor note... but it's more natural to put "width" first. X and Y, width and height - they usually go in that order. 

```suggestion
If the size of the UI component should be fixed, specify the [width](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#width) and [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height) properties.
```

File: concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+The UI component's height can also adapt to the UI component's contents. In this case, set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to `true`. To specify the minimum and maximum height that the adapted TextArea can occupy, specify the [minHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#minHeight) and [maxHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxHeight) properties.
```
Comment (Abadzhev) [inline concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md]:
```suggestion
The UI component's height can also adapt to the UI component's contents. In this case, set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to `true`. To specify the minimum and maximum height for the adaptive TextArea, use the [minHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#minHeight) and [maxHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxHeight) properties.
```

File: concepts/05 UI Components/TextArea/00 Getting Started with TextArea/20 Handle Value Change.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+When users type text into the TextArea, the UI component writes this text into the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property when the <a href="https://developer.mozilla.org/en/docs/Web/Events/change" target="_blank">change</a> event is raised. To specify another value change event, use the [valueChangeEvent](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#valueChangeEvent) property.
```
Comment (Abadzhev) [inline concepts/05 UI Components/TextArea/00 Getting Started with TextArea/20 Handle Value Change.md]: I removed the repetition of the word "when", shortened the text a bit, and got rid of the passive in the end.

```suggestion
If a user edits the text in a TextArea, the UI component updates the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property. The update happens on the <a href="https://developer.mozilla.org/en/docs/Web/Events/change" target="_blank">change</a> event. To specify another value change event, use the [valueChangeEvent](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#valueChangeEvent) property.
```

File: concepts/05 UI Components/Form/15 Configure Item Labels/05 Location and Alignment/05 Align Labels Relatively to Editors.md (hunk 2 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Form/15 Configure Item Labels/05 Location and Alignment/05 Align Labels Relatively to Editors.md:155]: I checked the code, it works for vertical alignment. Maybe the "h" represents something else...

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md [hunk 7]
```diff
@@ -63,7 +63,7 @@ The editor's width.
 ---
 The UI component offers users different editors to enter a value depending on the field's [dataType](/api-reference/_hidden/dxFilterBuilderField/dataType.md '/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/fields/#dataType'): [Calendar](/api-reference/10%20UI%20Components/dxCalendar '/Documentation/ApiReference/UI_Components/dxCalendar/'), [TextBox](/api-reference/10%20UI%20Components/dxTextBox '/Documentation/ApiReference/UI_Components/dxTextBox/'), [SelectBox](/api-reference/10%20UI%20Components/dxSelectBox '/Documentation/ApiReference/UI_Components/dxSelectBox/'), and so on. Use this function to customize those default editors or substitute them for other editors.
```
Comment (arminal) [inline api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md]: Can you please specify which UI component?
```suggestion
The UI component offers users different editors used to enter a value depending on the field's [dataType](/api-reference/_hidden/dxFilterBuilderField/dataType.md '/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/fields/#dataType'): [Calendar](/api-reference/10%20UI%20Components/dxCalendar '/Documentation/ApiReference/UI_Components/dxCalendar/'), [TextBox](/api-reference/10%20UI%20Components/dxTextBox '/Documentation/ApiReference/UI_Components/dxTextBox/'), [SelectBox](/api-reference/10%20UI%20Components/dxSelectBox '/Documentation/ApiReference/UI_Components/dxSelectBox/'), and so on. Use this function to customize those default editors or substitute them for other editors. 
```

File: concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
+If the size of the UI component should be fixed, specify the [width](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#width) and [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height) properties.
+
+The UI component's height can also adapt to the UI component's contents. In this case, set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to `true`. To specify the minimum and maximum height for the adaptive TextArea, use the [minHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#minHeight) and [maxHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxHeight) properties.
```
Comment (arminal) [inline concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md]:
```suggestion
The UI component's height can also adapt to UI component contents. In this case, set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to `true`. To specify the minimum and maximum height for the adaptive TextArea, use the [minHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#minHeight) and [maxHeight](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxHeight) properties.
```

**Pull request merged**